homeassistant/components/ring/manifest.json

```diff
@@ -13,5 +13,5 @@
   "documentation": "https://www.home-assistant.io/integrations/ring",
   "iot_class": "cloud_polling",
   "loggers": ["ring_doorbell"],
-  "requirements": ["ring-doorbell==0.7.3"]
+  "requirements": ["ring-doorbell==0.8.0"]
```

Comment:
Can you send this as a separate dependency bump? Makes it easier to review when the PR is doing fewer things.

Reply:
In this instance the dependency bump needs to go in at the same time as the code change.  This is because the new exception types in the ring_doorbell library are raised in the case of authentication errors and the oauthlib exceptions that the ring integration currently catches are no longer raised.  If the bump goes in without this change the existing authorisation flow in async_step_user will break.

Reply:
My assumption is that doing the bump and swapping exceptions is a little more mechanical (and can still preserve existing behavior) then actual reauth support is added second. I may have misunderstood though.

Reply:
Understood, that's now done, see https://github.com/home-assistant/core/pull/103904.